scheduler/cupsfilter.c

```diff
@@ -184,9 +184,11 @@ main(int  argc,				/* I - Number of command-line args */
 		i ++;
 		if (i < argc)
 		  num_options = cupsParseOptions(argv[i], num_options, &options);
-		else
+		else{
+      _cupsLangPrintf(stderr,"%s: Error - expected option after \"-a\" option.",argv[0]);
```

Comment:
Messages need to use the `_("string")` macro in order to get scanned by gettext (which we use to build the message catalog).